### ISSUE TITLE
Error NETWORK timeout -300

### ISSUE THREAD

**Issue by jondiez** [akamming/Garmoticz#2]
I have followed the steps of installing the widget and I have several rooms created in domoticz.  I put the configuration well because in the browser it works perfectly if I put the user, the password, the domain and the port.  I communicate via http as in the browser and I get NETWORK timeout -300 error

**Comment by ericguimont** [comment]
I thing you need to configur ssl accès in Domoticz . The widget use https access only.

**Comment by molusk** [comment]
I have the same issue and the reason seems to be the self-signed certificate :

> TLSv1.2 Record Layer: Alert (Level: Fatal, Description: Certificate Unknown)

As I am on a private network, it makes no sense to install a valid certificate.
Do you have any other solution ?

**Comment by GeitjeG** [comment]
I also use Domoticz via LAN or VPN, no SSL necessary. Making certificate is not a quick job. Hopefully Garmoticz can support plain http also!

**Comment by molusk** [comment]
I don't think Garmin allows plain http anymore and it seems they don't use android certificate store to check server's certificate.

I created an internal CA, pushed the CA cert into the android certificate store, then generated a certificate for domoticz. Checked the certificate was Ok with all the browsers I had to test (Edge and Firefox on my computer, Firefox only on my phone) and everything is Ok (if I install the CA certificate). But Garmoticz keeps failing with the same error (I didn't check the pcap file yet).

Anyway, even if it was working, this is really not a simple solution !
